tests/integration/update_cluster/docker-custom/in-v1alpha2.yaml

```
@@ -12,9 +12,9 @@ spec:
   containerRuntime: docker
   docker:
     packages:
-      urlAmd64: https://download.docker.com/linux/static/stable/x86_64/docker-20.10.1.tgz
+      urlAmd64: https://download.docker.com/linux/static/stable/x86_64/docker-20.10.5.tgz
```

This is meant to be an example about how to use a custom docker version. Could probably be any url and version there.
```suggestion
      urlAmd64: https://download.docker.com/linux/static/stable/x86_64/docker-20.10.1.tgz
```